errors.go

```diff
@@ -59,15 +63,16 @@ type ValidationFailedError struct {
 }
 
 func (e ValidationFailedError) Error() string {
-	msg := ""
+	msg := bytes.Buffer{}
+
 	for _, err := range e.APIError.err.Details.Errors {
 		if err.Name == "uniqueFieldIds" || err.Name == "uniqueFieldApiNames" {
-			return msg
-			// msg += err.Value["id"].(string) + " should be unique for " + err.Value["name"].(string) + "\n"
+			return msg.String()
```

Comment:
`msg.String()` will be an empty string here - it's also the case for current code base. I think error management requires a bit of refactoring. I'd like to keep this PR open for now until we have a better error handling in place.

Reply:
It's not necessarily empty, if a previous iteration of the loop encountered an error that is not caught in the if statement.

But I agree this part does need refactoring - this PR only solves a very specific issue I was having around one error.

Reply:
If you need this improvement now, I'm 👍 to merge. Otherwise I suggest to merge later since I'm currently working on a refactoring for this part. What do you think?

Reply:
If it's not a big problem, it would help to merge now. I want to add creation of entries and for sure I'll be hitting validation errors while doing that.